release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2892,3 +2892,24 @@ For more information, see xref:../updating/preparing-eus-eus-upgrade.adoc[Prepar
 ==== Updating
 
 To upgrade an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-10-15"]
+=== RHBA-2022:2258 - {product-title} 4.10.15 bug fix update
+
+Issued: 2022-05-23
+
+{product-title} release 4.10.15 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:2258[RHBA-2022:2258] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:2257[RHBA-2022:2257] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6960407[{product-title} 4.10.15 container image list]
+
+[id="ocp-4-10-15-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Image Registry Operator blocked installation on installer-provisioned infrastructure (IPI) IBM. With this update, clusters using manual credential minting will now require the administrator role. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2083559[*BZ#2083559*])
```

Comment:
Suggest revising 'manual credential minting' to 'clusters that mint credentials manually' or something similar for clarity.